ui/src/app/components/cases/steps-list.component.ts

```diff
@@ -370,5 +372,7 @@ export class StepsListComponent extends BaseComponent implements OnInit {
 
   setTestSteps($event: TestStep[]) {
     this.testSteps = $event;
+    let EnableStep=this.testSteps?.find(testStep=> !testStep.disabled)? true: false;
```

Comment:
1. plz use camelCase pattern for naming
2. plz recheck the enable step logic, try following, multiple steps with all disabled, one step not disabled, one step disabled, multiple step not disabled, multiple step with one disabled

Reply:
camelCase was done...above mentioned step logic also checked and working as expected.